.circleci/config.yml

```diff
@@ -301,6 +301,7 @@ jobs:
             for pkg in $(go list ./...)
```

Comment:
I will revert all changes in this file (config.yml) after review before merging. This was just to get a failing test quickly.